src/styles/_calendar.scss

```
@@ -0,0 +1,55 @@
+.calendar {
+  box-sizing: border-box;
+  padding: $padding;
+  width: ($padding * 2) + ($gap * 6) + ($size * 7);
+  display: flex;
+  align-items: center;
+  gap: $gap;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    width: $size;
+    height: $size;
+    background-color: $main-bg;
+    border: $border;
+    position: relative;
+    transition: all 0.5s;
+
+    &::before {
+      content: '';
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      top: 0;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    &:hover {
+      background-color: $hover-bg;
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+
+    @for $n from 1 through 31 {
+      &:nth-child(#{$n})::before {
+        content: '#{$n}';
+      }
+    }
+  }
+}
+
+@each $day, $margin in (mon: 0, tue: 1, wed: 2, thu: 3, fri: 4, sat: 5, sun: 6)
```

create a variable for it
```suggestion
@each $day, $margin in $weekDays
```